.goreleaser.yml

```diff
@@ -19,6 +19,8 @@ builds:
     goarch:
     - amd64
     - arm64
+    ldflags:
+    - '-s -w -X github.com/snyk/snyk-iac-rules/cmd.version={{.Version}}'
```

Comment:
The `-s -w` is copied from the default ldflags that were already being applied to releases: https://goreleaser.com/customization/build/